test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,136 @@
+@coolstore
+Feature: Cool Store
+  In order to test Minishift under load
+  I need to setup a test environment
+  Using CentOS and 4G of memory
+  After which I will import image streams
+  Finally I will deploy the Cool Store
+
+  Scenario: Enabling 'xpaas' add-on
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift addons enable xpaas"
+     Then stdout should contain
+      """
+      Add-on 'xpaas' enabled
+      """
+
+  Scenario: Starting Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --memory 4096 --iso-url centos" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: User can create new namespace 'coolstore'
+   Given Minishift has state "Running"
+    When executing "oc new-project coolstore" succeeds
+    Then stdout should contain
+     """
+     Now using project "coolstore"
+     """
+  
+  Scenario: User can create a template for coolstore
+   Given Minishift has state "Running"
+    When executing "oc create -f ./templates/coolstore.yaml" succeeds
+    Then stdout should contain
+     """
+     template "coolstore" created
+     """
+
+  Scenario: User can deploy the coolstore template to namespace 'coolstore'
+   Given Minishift has state "Running"
+    When executing "oc new-app --template=coolstore" succeeds
+    Then stdout should contain
+     """
+     Success
+     """
+
```

Comment:
Instead of this rather lengthy and imo ambigious rollout waiting, one could just add a new step:

    And services web-ui, catalog, cart rollout successfully

This way we are just saying that we expect the listed services to rollout. It avoids the ambiguity of implied order. Also it makes the test more concise. 

Internally one can still use the rollout command to watch the services. One can even do it "properly", reflecting what's really happening and go each watch in a go routine, waiting for each watch to complete/timeout/fail before moving on.